pkg/processor/sourceprocessor/attribute_filler.go

```diff
@@ -42,12 +42,12 @@ type attributeFiller struct {
 	labels          []string
 }
 
-func createSourceHostFiller() attributeFiller {
+func createSourceHostFiller(sourceHostAttrName string) attributeFiller {
```

Comment:
Just thinking out loud:
* perhaps we could add sourceHost as config option? (that would make it more streamline with other options)?
* should we use extractFormat() here instead of filling the struct inline?

Reply:
* you mean to provide an exact host name to be set as `_sourceHost`? As an alternative to get it from an attribute allowing both to be configured?
* I don't see the reason for that since filling this inline here works well enough. And using `extractFormat` might make it more complicated since we'll need to "bend" the logic to fit all source related templates 🤷

Reply:
Yeah, the only argument I have is consistency, I see your point though. That works